services/federator/src/Federator/Remote.hs

```diff
@@ -115,7 +115,7 @@ mkGrpcClient ::
   Members
     '[ Embed IO,
        Polysemy.Error RemoteError,
-       Polysemy.Reader TLSSettings
+       Polysemy.Reader (IORef TLSSettings)
```

Comment:
I'm not sure what you have in mind. If `TLSSettings` is just a pure value, how can we "change" it globally from another thread?

Reply:
Yeah, it can live in the `Env` type as an implementation detail, we don't expose that to the app. The app should always just be able to read it.

Reply:
Done in 51923d10b86b1702f138de24bd591dd1dc6751c0, but I went with an `Input` effect instead of `Reader`, for two reasons:
 1. It is a more straightforward first order effect, and we don't have to deal with `Local` when interpreting it
 2. There is already a ready-made function to interpret it by running an action every time